region/maridia/outer.json

```diff
@@ -5722,6 +5921,30 @@
                     }
                   ],
                   "devNote": "Can save an additional 4 energy with HiJump, and a bit more excess frames if not getting on top of the block to go to 2."
+                },
+                {
+                  "name": "G-Mode Morph",
```

Comment:
You can IBJ without grav from mama Turtle by bombing a baby turtle and springballing up to her back.